### ISSUE TITLE
Make User Join/Leave notifications available for unfocused sessions too

### ISSUE THREAD

**Issue by Banane9** [Yellow-Dog-Man/Resonite-Issues#1943]
### Is your feature request related to a problem? Please describe.

This was removed as a bug in #1933, but in my opinion it's quite handy to see when people join or leave a world you have open in the background. Especially for unfocused worlds, I'd want to know if someone I might want to talk to joins - or maybe someone I wanted to avoid left.

### Describe the solution you'd like

Add a distinguished notification type for unfocused sessions' join/leave notifications. These should be togglable in the Notification Settings.

### Describe alternatives you've considered

Mod it in again, I guess.

### Additional Context

The mod I had for the old platform used a less saturated color for the unfocused sessions' notifications, as well as a lowpass filtered version of the sound clip to evoke a "different room" effect.

### Requesters

Banane9

**Comment by troyBORG** [comment]
Yeah having the userjoin/leave toast show up but maybe in a different color to indicate it was for a different session.  Also a different sound effect for it as well.
